middlewares/expect-ct/index.ts

```diff
@@ -21,12 +21,12 @@ function parseMaxAge(value: void | number): number {
 function getHeaderValueFromOptions(options: Readonly<ExpectCtOptions>): string {
   const directives: string[] = [];
 
+  directives.push(`max-age=${parseMaxAge(options.maxAge)}`);
```

Comment:
This shouldn't be done as part of this pull request. Could you revert? We should also revert the changes in `test/expect-ct.test.ts`.

(It can also be done against the `master` branch instead of the `4.x` branch.)